modules/installation-configuration-parameters.adoc

```diff
@@ -1833,11 +1833,11 @@ a|Multiple IP addresses
 |String
 
 |`platform.vsphere.failureDomains.region`
-|You define a region by using a tag from the `openshift-region` tag category. The tag must be attached to the vCenter datacenter.
+|If you defined multiple failure domains for your cluster, you must attach the tag to each vCenter datacenter. You define a region by using a tag from the `openshift-region` tag category. You do not need to attach a tag for a single vSphere datacenter environment, but you must enter an alphanumeric value, such as `datacenter`, for the parameter.
```

Comment:
```suggestion
|If you define multiple failure domains for your cluster, you must attach the tag to each vCenter datacenter. To define a region, use a tag from the `openshift-region` tag category. For a single vSphere datacenter environment, you do not need to attach a tag, but you must enter an alphanumeric value, such as `datacenter`, for the parameter.
```

Reply:
I had to re-read that last sentence a few times before I understood what you meant. I thought beginning with an introductory phrase might help because it highlights the use case in which the user has to take the action you are describing.

Reply:
Thanks, @JoeAldinger . Good point. I updated the content. Please let me know what you think.